### ISSUE TITLE
Верстка нового дизайна

### ISSUE THREAD

**Issue by sergeivaskov** [bezumkin/orbita#13]
Сверстать по готовым макетам из Figma:
https://www.figma.com/design/YwhgUlzNYxzCkhN5SCncOL/%D0%9E%D1%80%D0%B1%D0%B8%D1%82%D0%B0?node-id=1-4&t=KVhlYG9yredUKvPW-1

Контакт дизайнера для связи в Telegram: @sergeivaskov

**Comment by bezumkin** [comment]
Если у кого-то есть желание попробовал улучшить стандартную тему Орбиты - вот неплохой вариант.

Лично мне, правда, больше нравится сайдбар с правой стороны, но передвинуть не проблема. А может, даже, и настройкой его сделать.

В общем, желающим поверстать на досуге - добро пожаловать!

**Comment by sergeivaskov** [comment]
@bezumkin опишу почему выбраны данные решения:

> больше нравится сайдбар с правой стороны

В данном случае исходил из стандартной логики повествования, что первым делом пользователю нужно представиться, дать понять туда ли он попал, показав логотип, название и описание. Но в целом можно действительно сделать опциональным перенос сайдбара на правую сторону, возможно кто-то окажется полезным.

> насчёт меньшей ширины текста в заметке я не уверен

Изначально я исходил из ширины контента на главной странице, чтобы не возникало ситуации, при которой сильно увеличивалось бы изображение превью, если использовать стандартное соотношение сторон 16:9. При меньшем размере поста по ширине на маленьких экранах помещается как превью, так и текстовое описание.

Затем я использовал эту же ширину и для основного поста. Удобство небольшой ширины контента обусловлено банальной простотой перехода глазами между строками. Чем уже контент, тем проще глазами найти начало следующей строки.

**Comment by bezumkin** [comment]
@sergeivaskov Если что, я не забыл - просто времени нет.

Дизайн мне очень нрайится и обязательно пойдёт в работу, просто непонятно, когда именно.

**Comment by sergeivaskov** [comment]
@bezumkin да, без проблем. Я на связи. Будет время, буду дополнять прочими элементами по возможности.

**Comment by chemist-repo** [comment]
@bezumkin @sergeivaskov а макет уже в работе?

**Comment by bezumkin** [comment]
@chemist-repo Нет, никто не стал делать.

Думаю, оформление по умолчанию останется как есть сейчас.
